ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -934,8 +934,6 @@ static const func_info_t func_infos[] = {
 	F0("eregi",			                        MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_LONG),
 	F1("eregi_replace",	                        MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_STRING),
 	F1("split",			                        MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_LONG | MAY_BE_ARRAY_OF_STRING),
```

Comment:
We can drop these functions from `/* ext/ereg */` as well

Reply:
done